packages/api/tests/models/granules/test-granule-reporting.js

```diff
@@ -253,6 +267,96 @@ test('_storeGranuleRecord() will allow a running status to replace a failed stat
   );
 });
 
+test('_storeGranuleRecord() will allow a completed status to replace a queued status for a new execution', async (t) => {
```

Comment:
This test is written correctly according to the test name, but I don't think this is a valid test case. The expected case I think is that a "completed" status should be able to replace a "queued" status for the same execution, not a new one.